src/openrct2/ride/Vehicle.cpp

```diff
@@ -86,12 +86,14 @@ static SoundId vehicle_update_scream_sound(rct_vehicle* vehicle);
 static void vehicle_kill_all_passengers(rct_vehicle* vehicle);
 static bool vehicle_can_depart_synchronised(rct_vehicle* vehicle);
 
-#define VEHICLE_INVALID_ID (-1)
+constexpr int VEHICLE_INVALID_ID = -1;
```

Comment:
int32_t

Reply:
Addressed.
@duncanspumpkin Do you mean everywhere that addresses `trainIndex` should be refactored to `int32_t`? I can do that on a follow-up PR

Reply:
yeah.